libs/metrics/base.py

```diff
@@ -194,7 +195,9 @@ def compute_live_metrics(cls, config, live_interval):
                 return
             entity["base_directory"] = entity_directory
             entries_in_interval = int(live_interval * 60 / get_source_logging_interval(config))
-            live_data = cls.generate_live_csv_data(today_entity_csv, entity, entries_in_interval)
+            if not cls.can_execute(config, entity):
```

Comment:
This condition, should be also checked  in the hourly, daily and weekly methods?

Reply:
I presume yes, my intent was on doing those on a separate PR an see but I could add them in this very same PR